Students/Semester2/lessons/students/2_functions_and_conditions/12_advanced_conditionals/12_advanced_conditionals.ipynb

```diff
@@ -175,10 +175,21 @@
   }
  ],
  "metadata": {
+  "kernelspec": {
```

Comment:
The files that have changes like these *dont have any actual code changes* - you just *ran the jupyter notebook* on your computers. Since the original author (probably me) ran it on a different python version (3.9.x) and you ran it below at 3.10.6, it looks like some code change happens.

The short term fix is we just don't commit these changes in files where there aren't any actual code changes. The jupyter notebook versions would just change depending on the last contributor of __real code__ into the lesson.

We should do a long term fix here. I think if we have everyone contributing to this repo running the same version of Python (for example the one that you're using `"Python 3.10.6 64-bit"` but we can always pick a newer version) then no matter who is contributing these fields won't change. We should specify this Python version in the top level `README.md`